templates/base/src/stores/global/tests/actions.test.js

```diff
@@ -0,0 +1,17 @@
+import * as globalActions from './../actions';
+import { UPDATE_APP_ERROR } from './../constants';
+
+describe('Testing Global Actions JS File', () => {
+    let payload = { errorInfo: {}, isError: true };
+
+    test('Global Actions is defined', () => {
+        expect(globalActions.updateAppError).toBeDefined();
+    });
+
+    test('Should dispatch updateAppError', () => {
+        expect(globalActions.updateAppError()).toEqual({
```

Comment:
This call to globalActions.updateAppError() should have payload  and mocked dispatch as its parameters